services/models/service.go

```diff
@@ -6,8 +6,18 @@ import (
 )
 
 type ServiceOverview struct {
-	Name         string `json:"name"`
-	IstioSidecar bool   `json:"istioSidecar"`
+	// Name of the workload
+	// required: true
+	// example: reviews-v1
+	Name string `json:"name"`
+	// Has IstioSidecar
```

Comment:
If this is for swagger, it should have a little bit more workding to not confuse users.
It can say something like "Define if Pods related to this Service has an IstioSidecar deployed".

Just to avoid the confusion that a Sidecar is not par of a Service per se.